make-webpack-config.js

```diff
@@ -130,11 +134,12 @@ module.exports = function(options) {
             path.resolve(__dirname, 'src/frame/'),
             path.resolve(__dirname, 'src/shared/')
         ],
-        use: [
-            'style-loader',
-            'css-loader',
-            'less-loader'
-        ]
+        use: ExtractTextPlugin.extract({
```

Comment:
this is how you tell webpack to generate and external file for this type of module. In this case, we're telling it to generate a css file for css modules it encounters instead of inlining it inside the js bundle. You could do the same for html files if you wanted for instance.